README.md

```diff
@@ -76,19 +94,55 @@ To improve build times you can add the following options to the command:
 | `-Dspotless.check.skip=true`  | Skip the Spotless code style checks                 |
 | `-o`                          | Work offline so Maven does not download any updates |
 | `-T 1C`                       | Build in parallel, using 1 thread per core          |
+| `-pl :<add-on directory>`     | Build a single add-on                               |
 
 For example you can skip checks and tests during development with:
 
-`mvn clean install -DskipChecks -DskipTests`
+```shell
+mvn clean install -DskipChecks -DskipTests -pl :org.openhab.binding.astro
+```
 
 Adding these options improves the build time but could hide problems in your code.
 Parallel builds are also less easy to debug and the increased load may cause timing sensitive tests to fail.
 
-To check if your code is following the [code style](https://www.openhab.org/docs/developer/guidelines.html#b-code-formatting-rules-style) run: `mvn spotless:check`
-To reformat your code so it conforms to the code style you can run: `mvn spotless:apply`
+#### Translations
+
+Add-on translations are managed via [Crowdin](https://crowdin.com/project/openhab-addons).
+The english translation is taken from the openHAB-addons GitHub repo and automatically imported in Crowdin when changes are made to the english i18n properties file.
+When translations are added or updated and approved in Crowdin a pull request is automatically created by Crowdin.
+Therefor translations should not be edited in the openHAB-addons repo, but only in Crowdin.
+Otherwise translation are overridden by the automatic process.
+
+To fill the english properties file run the following maven command on an addon-on:
+
+```shell
+mvn i18n:generate-default-translations
```

Comment:
The command is not working for everybody. 
The following is apparently working for everybody: mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations

Reply:
I've added the command you mentioned here as an alternative option in the text below in case above command doesn't work.